src/main/java/seedu/address/logic/commands/EditPolicyCommand.java

```diff
@@ -30,7 +30,7 @@ public class EditPolicyCommand extends Command {
             + ": Updates the specified policy for the client identified "
             + "by the index number used in the last client listing. \n"
             + "Parameters: INDEX (must be a positive integer) "
-            + "pt/[POLICY_TYPE] pa/[PREMIUM_AMOUNT] ca/[COVERAGE_AMOUNT] ed/[EXPIRY_DATE]\n"
+            + "pt/POLICY_TYPE [pa/PREMIUM_AMOUNT] [ca/COVERAGE_AMOUNT] [ed/EXPIRY_DATE]\n"
```

Comment:
we should use the statically defined prefix here as well as it makes the code for adaptable to changes. (eg. if we change the statically defined prefix in the future, this wont be affected)

Reply:
Updated again!